### PR TITLE
Add opam file for HoTT library for Coq 8.16

### PR DESCRIPTION
**Pull request by Alizter**

### CODE REVIEW

File: released/packages/coq-hott/coq-hott.8.16/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description: """
+To use the HoTT library, the following flags must be passed to coqc:
+   -noinit -indices-matter
+To use the HoTT library in a project, add the following to _CoqProject:
+   -arg -noinit
+   -arg -indices-matter
+"""
+tags: [ "logpath:HoTT" ]
+url {
+  src: "https://github.com/HoTT/HoTT/archive/refs/tags/V8.14.tar.gz"
```
Comment (silene) [inline released/packages/coq-hott/coq-hott.8.16/opam:27]: 8.14?

Comment (Alizter) [reply released/packages/coq-hott/coq-hott.8.16/opam]: Fixed

**Comment by Alizter** [comment]
I generated the sha512 by hand, but I am always a little frightened if I did it correctly. What is the way to check?

**Comment by palmskog** [comment]
@Alizter what exactly does "by hand" mean? Is it different from running `sha512sum <my-file>`?

**Comment by palmskog** [comment]
I think @MSoegtropIMC wants to know about this one.

**Comment by silene** [comment]
> I generated the sha512 by hand, but I am always a little frightened if I did it correctly. What is the way to check?

I usually run `opam lint --check-upstream path/to/opam`.

**Comment by Alizter** [comment]
> @Alizter what exactly does "by hand" mean? Is it different from running `sha512sum <my-file>`?

Yes that is what I mean. I was wondering if opam was supposed to do this for me or something.

> > I generated the sha512 by hand, but I am always a little frightened if I did it correctly. What is the way to check?
> 
> I usually run `opam lint --check-upstream path/to/opam`.

Thanks!

**Pull request merged**